spec/services/file_upload/file_uploader_spec.rb

```diff
@@ -10,12 +12,77 @@
 
     context 'production environment' do
       before do
-        allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new('production'))
+        allow(Rails).to receive_message_chain(:env, :production?).and_return(true)
```

Comment:
I really don't like this... Not your change, but rather the whole context here. Using Rails.env.production is a bit iffy in general, as it rather goes against the general 12-factor approach that an app shouldn't know its own environment. And then _overriding_ that... Yeesh. If we want to sometimes use AwsFileUploader in the test environment, then the logic to decide whether to use AwsFileUploader shouldn't be controller with Rails.env.production.

But this might be a battle for another day...